stix_shifter_modules/azure_log_analytics/configuration/lang_en.json

```diff
@@ -0,0 +1,44 @@
+{
+  "connection": {
+    "host": {
+      "label": "Management IP address or Hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+    },
+    "port": {
+      "label": "Host Port",
+      "description": "Set the port number that is associated with the Host name or IP"
+    },
+    "help": {
+      "label": "Need additional help?",
+      "description": "More details on the data source setting can be found in the specified link"
+    },
+    "selfSignedCert": {
+      "label": "Microsoft Azure Sentinel Certificate",
+      "description": "Use SSL certificate for Microsoft Azure Sentinel."
+    },
+    "sni": {
+      "label": "Server Name Indicator",
+      "description": "The Server Name Indicator (SNI) enables a separate hostname to be provided for SSL authentication."
+    },
+    "workpaceId": {
+      "label": "Log Analytics Workspace ID",
+      "description": "The workspace Id of the Log Analytics. Required when using Log Analytics API. Ignore if using Graph Security."
```

Comment:
Does "Graph Security" need to be here? Isn't that the "old" azure sentinel connector?

Reply:
no. inherited from previous pr. removing it.